react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import hotButton from 'hot-button';
+
+let counter = 1;

```

Comment:
This is not a good idea, it is like using a global variable and could cause weird side effects